demos/appyx-interactions/android/src/main/kotlin/com/bumble/appyx/interactions/sample/ModalExperiment.kt

```diff
@@ -0,0 +1,110 @@
+package com.bumble.appyx.interactions.sample
+
+import androidx.compose.animation.core.Spring
+import androidx.compose.animation.core.spring
+import androidx.compose.foundation.gestures.detectDragGestures
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.unit.dp
+import com.bumble.appyx.components.modal.Modal
+import com.bumble.appyx.components.modal.ModalModel
+import com.bumble.appyx.components.modal.ui.ModalMotionController
+import com.bumble.appyx.interactions.AppyxLogger
+import com.bumble.appyx.interactions.core.ui.context.UiContext
+import com.bumble.appyx.interactions.core.ui.gesture.GestureSettleConfig
+import com.bumble.appyx.interactions.core.ui.helper.InteractionModelSetup
+import com.bumble.appyx.interactions.sample.android.Children
+import com.bumble.appyx.interactions.sample.android.Element
+import com.bumble.appyx.transitionmodel.BaseMotionController
+import com.bumble.appyx.interactions.sample.InteractionTarget as Target
+
+@ExperimentalMaterialApi
+@Composable
+@Suppress("LongMethod", "MagicNumber")
+fun ModalExperiment(
+    modifier: Modifier = Modifier,
+    motionController: (UiContext) -> BaseMotionController<Target, ModalModel.State<Target>, *, *>
+) {
+    val items = listOf(
+        Target.Child1,
+        Target.Child2,
+        Target.Child3,
+        Target.Child4,
+        Target.Child5,
+        Target.Child6,
+        Target.Child7,
+        Target.Child1,
+        Target.Child2,
+        Target.Child3,
+        Target.Child4,
+        Target.Child5,
+        Target.Child6,
+        Target.Child7,
+        Target.Child1,
+        Target.Child2,
+        Target.Child3,
+        Target.Child4,
+        Target.Child5,
+        Target.Child6,
+        Target.Child7,
+    )
+    val modal = Modal(
+        model = ModalModel(
+            initialElements = items,
+            savedStateMap = null
+        ),
+        motionController = motionController,
+        gestureFactory = { ModalMotionController.ModalGestures(it) },
+        animationSpec = spring(stiffness = Spring.StiffnessVeryLow / 4),
```

Comment:
Right now this line has no effect, since you made your operations to be run `KEYFRAME` as a default. This line only affects `IMMEDIATE` ones. If you do change the operation, you will start seeing this very low stiffness having effect. I'd just suggest removing this line altogether.

Reply:
Done